packages/agreement-process/src/model/domain/validators.ts

```diff
@@ -380,6 +365,15 @@ const attributesSatisfied = <
   });
 };
 
+export const failOnActivationFailure = (
```

Comment:
Moved this here from below because this is not a filter/matcher but an assertion (it throws)